### ISSUE TITLE
Failed to activate the language-javascript-semantic package

### ISSUE THREAD

**Issue by igorshubovych** [p-e-w/language-javascript-semantic#18]
Steps to reproduce below:
1. Open the app

**Atom Version**: 0.194.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [language-javascript-semantic](https://github.com/p-e-w/language-javascript-semantic) package, v0.1.0

### Stack Trace

Failed to activate the language-javascript-semantic package

```
At Cannot read property 'addGrammar' of undefined

TypeError: Cannot read property 'addGrammar' of undefined
    at Object.module.exports.activate (/Users/igor/.atom/packages/language-javascript-semantic/lib/main.coffee:8:25)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:195:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:418:29
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:311:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:413:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:394:46)
    at Atom.module.exports.Atom.startEditorWindow (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:623:21)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:12:8)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:23:4)
    at Module._compile (module.js:452:26)
    at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
    at Module.load (module.js:347:32)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "unity-ui",
      "base16-tomorrow-light-theme"
    ],
    "excludeVcsIgnoredPaths": false,
    "disabledPackages": [
      "atom-ctags"
    ]
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.1.0
atom-jasmine, v0.5.0
autocomplete-html, v0.5.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.12.0
autocomplete-ruby, v0.1.0
autocomplete-snippets, v1.2.0
color-picker, v1.7.0
command-logger, v0.23.0
comment, v0.2.7
copy-as-rtf, v0.6.0
editorconfig, v1.0.0
file-icons, v1.5.4
javascript-snippets, v1.0.0
language-javascript-semantic, v0.1.0
linter, v0.12.1
minimap, v4.7.6
project-manager, v1.15.5
remote-sync, v2.4.1
save-session, v0.14.0
unity-ui, v2.0.7

# Dev
No dev packages
```

**Comment by tomwganem** [comment]
+1

**Comment by p-e-w** [comment]
Fixed in v0.2.1. Please update.
